index.html

```diff
@@ -317,12 +334,12 @@ <h3 class="form-h3">Get started</h3>
             </li>
           </ul>
           <div class="get-my-cv button-bottom">
-            <button
-              class="get-my-cv-btn start-colab colab-butn colab-detail"
-              type="submit"
+            <a
+              href="#"
+              class="get-my-cv-btn main-button button-action last-button"
             >
               Start Collaboration
-            </button>
+            </a>
```

Comment:
[ Required Changes ]

- [ ] Kindly make this a button tag or an input tag with a type of button and a value. The form is not being submitted to Form Spree because you're using an `a` tag which is meant for linking pages. The button stage is meant for submitting form data.
**Hint:**
```mdx
<button type="submit"> Start Collaboration </button>
```

- [ ] You did all other requirements for this project on the form section and when I removed the `a` tag and used button tag, the form data was submitted successfully. Please fix that 👍 in order for your project to be complete.